### ISSUE TITLE
GG Max - Latest EXP build - Enemy Character does not even walk a path, and keeps teleporting backwards after a few steps

### ISSUE THREAD

**Issue by Tum1370** [TheGameCreators/GameGuruRepo#5041]
As the title says, I did a reset of my Gameguru folder, setup a simple map to test the loot drops, but when i placed 1 character, and several flags, linked them all together, so the enemy would randomly pick a flag to walk to.
The character just walks forward a couple of steps then teleports back to where it was.

I have included the simple map for this test.

[TEST.zip](https://github.com/TheGameCreators/GameGuruRepo/files/13239108/TEST.zip)

**Comment by Tum1370** [comment]
Also another bug, I verified all MAX files. 
I deleted my Max Folder in My Documents, and let Max create a new one, so everything is fresh.
I start a NEW project.
Click to Create the Level, and i do not get taken to the Terrain Creator, And Max just seems to FREEZE.
I then Shut Max down, Restart Max, Then i click the level to go to Terrain Generator, and it works.

But after creating a very simple terrain, i get the bug that i have reported here.

Very disappointing, after seeing all these bug fixes, coming back to very simple bugs like this.
Makes me wonder how many more bugs i will find if i actually look.
